bin/deploy.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+echo "Running deploy script"
+export DEBIAN_FRONTEND=noninteractive
+echo "localdev" > /etc/hostname
+echo "127.0.0.1 localdev" >> /etc/hosts
+
+echo "Install essential software pacakges"
+apt-get -qq update
+apt-get -qq install -y git awscli openssl libssl-dev
+
+echo "Configure AWS SQS access"
+
+if [[ "$SQS_IDENT" != _* ]]
+	then
+	echo "ERROR! Missing or malformed critical environment variables: SQS_IDENT"
+	exit 1
+fi
+if [[ -z "$AWS_REGION" || -z "$AWS_ACCESS_KEY_ID" || -z "$AWS_ACCESS_SECRET" ]]
+    then
+	echo "ERROR! Missing critical environment variable: AWS_REGION, AWS_ACCESS_KEY_ID, AWS_ACCESS_SECRET!"
+    exit 1
+fi
```

Comment:
For the AWS stuff, yes! For `SQS_IDENT`, no. That is because the packer script that creates AMIs doesn't need that variable, and therefore won't populated it. It's only used for local development.